src/lib/hypertune/hypertune.ts

```diff
@@ -0,0 +1,19 @@
+import { PUBLIC_HYPERTUNE_TOKEN } from '$env/static/public';
+import { initializeHypertune } from './generated';
+
+const hypertune = initializeHypertune({}, { token: PUBLIC_HYPERTUNE_TOKEN });
+
+export async function getRootNode(user: { id: string; email?: string }) {
+	await hypertune.initFromServerIfNeeded();
+
+	return hypertune.root({
+		context: {
+			environment: 'DEVELOPMENT',
```

Comment:
is this supposed to be development

Reply:
Doesn't matter you can activate/deactivate them based on the env but we don't currently so I don't think it matters.